deploy/mariadb/templates/backuppolicytemplate.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: BackupPolicyTemplate
+metadata:
+  name: mariadb-backup-policy-template
```

Comment:
missing a clusterDefinition labels, `clusterdefinition.kubeblocks.io/name: mariadb`.
now, the controller backupPolicyTemplate has not been implemented yet. We need to manually add the label for ClusterDefinition to establish relationships.

Reply:
thanks, it works !